ui-permissions/RM-super-user-permissions.sql

```diff
@@ -0,0 +1,17 @@
+insert into casev3.user_group
+values ('8269d75c-bfa1-4930-aca2-10dd9c6a2b42', 'Super user - full permissions', 'super');
+
+insert into casev3.user_group_permission
+values ('c469377e-680e-4cb1-92a0-5217be2b3a52', -- permission ID
+        'SUPER_USER',                           -- activity
+        '8269d75c-bfa1-4930-aca2-10dd9c6a2b42', -- group ID
+        null);
+
+insert into casev3.users
```

Comment:
I think this should be the 1st command and have some bad SQL along the lines of
<INSERT YOUR ONS EMAIL ADDRESS HERE> NOT a String.

Then it will fail if you don't change it.  As it would be the 1st command it would make rerunning it all easier